tests/decodeinputavformat.cpp

```diff
@@ -38,11 +38,11 @@ bool DecodeInputAvFormat::initInput(const char* fileName)
         goto error;
     uint32_t i;
     for (i = 0; i < m_format->nb_streams; ++i) {
-        AVCodecContext* ctx = m_format->streams[i]->codec;
```

Comment:
After realizing that the deprecated parameter was introduced on latest libavcodec version, I think it is good to support at least current release version and one before.